src/app/pages/things/details/things.details.component.ts

```diff
@@ -30,6 +27,12 @@ export class ThingsDetailsComponent implements OnInit, OnDestroy {
   selectedChannels = [];
   editorMetadata = '';
 
+  filters: MsgFilters = {
+    offset: 0,
+    limit: 20,
+    subtopic: '',
+  };
+
```

Comment:
Maybe you should explicitly set other fields to zero values. It's more readable like that.

Reply:
done